### DISCUSSION TITLE
Windows App SDK 1.6.1 (1.6.240923002)

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#4766] [Announcements]
## Windows App SDK 1.6.1 (1.6.240923002)

Windows App SDK 1.6.1 is a stable servicing release that fixes critical bugs for 1.6.

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.6 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://aka.ms/windowsappsdk/1.6/1.6.1/release-notes) for the list of issues fixed in this release.

### Updating to this version

The 1.6.1 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.6.240923002

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.6.1'>Windows App SDK 1.6.1 (1.6.240923002)</a>.</em>

**Comment by ghost1372** [comment]
@duncanmacmichael 
Can we expect a new version this Wednesday? (Almost two months have passed since the last version)
**v1.7-exp1** and v1.6.**2**

**Reply by duncanmacmichael** [reply]
Just released, thanks for your patience!
